e2e/pierced-react-remix-fragment/app/root.tsx

```diff
@@ -39,6 +39,16 @@ export function Layout({ children }: { children: React.ReactNode }) {
 				<ScrollRestoration />
 				<Scripts />
 			</body>
+			{/* 
+          This is a hack to work around an issue with WritableDOM not populating
+          the contents of the last script element (which just so happens to be responsible
+          for hydrating the application). We noticed that if that script tag wasn't the last
+          child of the body, its contents were inserted correctly. We're adding this empty
+          div here to make the script tag before it no longer be the last child.
+
+          TODO: https://jira.cfdata.org/browse/DEVDASH-79
```

Comment:
do we want internal links here?

also is this actually still an issue? didn't we find out this wasn't really the problem or something? (but I might misremember things...)

Reply:
You're right, no we do not want references to internal documentation. I'll clean up the hack div, rebase and push

Reply:
yeah, this doesn't seem to be an issue here. here you can see the final inline script tag has its text contents populated correctly already.
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/cc607e1a-fe45-4929-a6e3-3f92caa4a32a">

Reply:
But in the current demo, we are reframing from target, so we don't use writable-dom at all. When reframing with fetch, that's when I see the issue